.github/workflows/aro-hcp-dev-env-cd.yml

```diff
@@ -262,6 +262,11 @@
             sudo mv oc /usr/local/bin/oc
             chmod +x /usr/local/bin/oc
 
+        - uses: azure/setup-helm@v4.2.0
+          with:
+            version: 'v3.13.3'
+          id: install
+
```

Comment:
This should go a bit further down, in the management cluster section since we're deploying maestro agent right?